dotCMS/src/main/java/com/liferay/portal/ejb/UserPersistence.java

```diff
@@ -49,30 +54,34 @@
  *
  */
 public class UserPersistence extends BasePersistence {
+
+	private static final String SEARCH_USER_BY_ID="select * from user_ where userid=?";
+	private static final UserTransformer userTransformer = new LiferayUserTransformer();

```

Comment:
this should follow the constants name style

Reply:
the userTransformer is not a string is a class and I'm using the the style https://google.github.io/styleguide/javaguide.html#s5.2.4-constant-names